test/studies/IMSuite/vertexColor/vertexColoring-serial.chpl

```diff
@@ -0,0 +1,319 @@
+ /* This file is part of IMSuite Benchamark Suite.
+
+    This file is licensed to You under the Eclipse Public License (EPL);
+    You may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.opensource.org/licenses/eclipse-1.0.php
+
+   (C) Copyright IMSuite 2013-present.
+ */
```

Comment:
Hi Kushal,

After some discussion on our end, I don't think there are any problems having an EPL-licensed file in the repository. After a couple of changes, I can merge this in:

1. Remove the line "This file is part of IMSuite Benchamark Suite.", because it's not actually part of the official IMSuite.
2. Add a link to the IMSuite website noting that anyone interested can see the original program there.

Thanks!